tests/benchmark/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM debian:bullseye
```

Comment:
We can consider using Alpine since it's a more lightweight image - less prone to security issues.

Reply:
I think using our AP-airflow image will make it MUCH faster as we won't need to build any of this from scratch.

Reply:
Is this an improvement we could do in a separate PR?